### PR TITLE
HWC-6

### PR DESCRIPTION
**Pull request by HelplightningXeon**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ module.exports = {
   mode: isDev ? 'development' : 'production',
 
   entry: {
-    "HelpLightningClient": ['./src/core/HelpLightningClient.js']
+    "HelpLightningClient": ['./src/core/HelpLightningClient.js'],
+    "invite": ['./src/core/features/invite/components/Invite']
```
Comment (mdillavou) [inline webpack.config.js]: I think we should make this:
```
    "components/Invite": ['./src/core/features/invite/components/Invite']
```
Then you'd do something like:
```
import {Invite} from 'components/Invite.js'
```
right?

Comment (HelplightningXeon) [reply webpack.config.js]: There is a way to export the modules that user can import like below.
```
import { HelpLightningClient, components } from 'helplightning-web-components'; 
const { Invite } = components;
```

By our current export approach, user will import components like,
import Invite from 'helplightning-web-components/dist/components/Invite'.
Please note the 'dist' in the path.

Which style do you prefer? @mdillavou

Comment (HelplightningXeon) [reply webpack.config.js:9]: By our current export approach, user will import components like,
`import Invite from 'helplightning-web-components/dist/components/Invite'`.

Comment (mdillavou) [reply webpack.config.js]: ok, I prefer the current `import Invite from 'helplightning-web-components/dist/components/Invite`. It'd be great if we eventually submitted this to npmjs. Do you know how we'd get rid of the dist? Is it just changing how we pack it?

**Comment by mdillavou** [comment]
Merged in master in cf7b0f5c8d168f12af581d1cc402797c2ad3e144

**Pull request merged**